scripts/sizeSnapshot/create.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-console */
+const childProcess = require('child_process');
+const fse = require('fs-extra');
+const { flatten, fromPairs } = require('lodash');
```

Comment:
The downside of using named import is the friction it's creating when editing the code. We have to go to the top of the file to add the missing dependencies. What do you think of using a different tradeoff for the server side script, where we don't care about the bundle size?

```suggestion
const lodash = require('lodash');
```

Reply:
Makes sense to me. Will change